qiling/os/posix/syscall/stat.py

```diff
@@ -319,6 +319,57 @@ class LinuxMips64Stat(ctypes.Structure):
 
     _pack_ = 8
 
+class LinuxMips32EBStat(ctypes.BigEndianStructure):
```

Comment:
hmmm, where does this structs come from? Could you add a few docs?

Reply:
It is the same with MIPS little endian, which was already documented at

https://github.com/qilingframework/qiling/blob/2b47ec76fcb7fb64cc9c3f562ce64968f4b0303d/qiling/os/posix/syscall/stat.py#L187-L320

Since there is no `#if defined(__ARMEB__)`-ish thing in this file, it indicates that both MIPS little endian and big endian share the same structs.